### PR TITLE
JSON Appsettings refactor

### PR DESCRIPTION
**Pull request by tuday2**
Issue #209
Submitting for initial review, still have more error messages to add

### CODE REVIEW

File: MBBSEmu/Configuration.cs [hunk 1]
```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.IO;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace MBBSEmu
+{
+    public static class AppSettings
```
Comment (enusbaum) [inline MBBSEmu/Configuration.cs:9]: Let's not make this static, make it an instanced class that we can pass into classes using Dependency Injection

File: MBBSEmu/Configuration.cs [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        public static bool RloginPortPerModule => GetBoolAppSettings("Rlogin.PortPerModule");
+        public static string DatabaseFile => GetStringAppSettings("Database.File");
+        
+        //Optional Keys
+        public static string ANSILogin => ConfigurationRoot["ANSI.Logon"];
+        public static string ANSILogoff => ConfigurationRoot["ANSI.Logoff"];
+        public static string ANSISignup => ConfigurationRoot["ANSI.Signup"];
+        public static string ANSIMenu => ConfigurationRoot["ANSI.Menu"];
+
+        private static string GetStringAppSettings(string key)
```
Comment (enusbaum) [inline MBBSEmu/Configuration.cs:49]: We can probably combine these methods using a `GetAppSettings<T>(string key)`

**Review by enusbaum — CHANGES_REQUESTED** [review]

File: MBBSEmu/Configuration.cs (hunk 1 above)
Comment (tuday2) [reply MBBSEmu/Configuration.cs]: Fixed.

**Comment by tuday2** [comment]
Ready for review

**Review by enusbaum — APPROVED** [review]

**Pull request merged**